pkg/platform/abstract/platform.go

```diff
@@ -1686,11 +1686,35 @@ func (ap *Platform) validateTriggers(functionConfig *functionconfig.Config) erro
 				}
 			}
 		}
+
+		if err := ap.validateBatchConfiguration(triggerInstance.Batch, functionConfig.Spec.Runtime, triggerInstance.Kind); err != nil {
+			return nuclio.WrapErrBadRequest(err)
+		}
 	}
 
 	return nil
 }
 
+func (ap *Platform) validateBatchConfiguration(batchConfiguration *functionconfig.BatchConfiguration, runtime, triggerKind string) error {
```

Comment:
Since you're logging and not only returning a bad request, you need the context object.
Also, you can pass the entire function config to make it a bit cleaner.
```suggestion
func (ap *Platform) validateBatchConfiguration(ctx context.Context, functionConfig *functionconfig.Config) error {
```